packages/typespec-ts/test/commands/run.ts

```diff
@@ -0,0 +1,224 @@
+import { fileURLToPath } from "url";
+import { dirname, join as joinPath } from "path";
+import {
+  Extractor,
+  ExtractorConfig,
+  ExtractorLogLevel,
+  IExtractorConfigPrepareOptions
+} from "@microsoft/api-extractor";
+import { npxCommand } from "./runCommand.js";
+import * as fs from "fs/promises";
+import { Dirent } from "fs";
+import { createTaskLogger } from "./logger.js";
+import { createProgram } from "typescript";
+
+export async function runTypespec(config: any, mode: any) {
+  await run(createContext());
```

Comment:
nit: I was getting confused thinking that run was trying to run a task to create the context, may be clearer if you create the context first and then pass it to the run function